### PR TITLE
VoiceChannel fixes

### PR DESCRIPTION
**Pull request by Monti934**
- Back to teams button will disconnect user from voice chat
- breakout rooms for seperate tasks with the same name will now create seperate voice channels
- on joining a channel, a loading screen is shown which prohibits user input to avoid errors (e.g. for long response time of agora or wait to allow microphone usage)
- setting a Task to In_Session_Done will now remove the corresponding voicechat from the list. However the users will not be kicked out, a room is closed when the last member leaves. On reactivating the task, the voice channel will be reinstantiated aswell (added a TODO comment for pusher for this and need the updated pusher without location.reload to work)

### CODE REVIEW

File: src/components/ui/VoiceChat.tsx [hunk 1]
```diff
@@ -8,12 +8,41 @@ import AgoraRTM from "agora-rtm-sdk"; //RTM for Channels, Users, etc.
...
+        height: "40px",
+        border: "4px solid #f3f3f3",
+        borderTop: "4px solid #3498db",
+        borderRadius: "50%",
+        animation: "spin 1s linear infinite",
+      }}
+    />
+  </div>
+);
+
```
Comment (5v3nn) [inline src/components/ui/VoiceChat.tsx:38]: make separate component and replace every copy-pasted creation of that as well.

**Review by 5v3nn — CHANGES_REQUESTED** [review]

File: src/components/ui/VoiceChat.tsx (hunk 1 above)
Comment (Monti934) [reply src/components/ui/VoiceChat.tsx]: done, others will be replaced in different branch

**Review by 5v3nn — APPROVED** [review]
ok

File: src/components/ui/VoiceChat.tsx (hunk 1 above)
Comment (5v3nn) [reply src/components/ui/VoiceChat.tsx]: thank you

**Pull request merged**